server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -329,15 +465,33 @@ void addNodes(int newNodesCount) {
         }
 
         void stabilise() {
-            stabilise(DEFAULT_STABILISATION_TIME);
+            stabilise(DEFAULT_STABILISATION_TIME, DEFAULT_DELAY_VARIABILITY);
         }
 
         void stabilise(long stabiliationDurationMillis) {
+            stabilise(stabiliationDurationMillis, DEFAULT_DELAY_VARIABILITY);
+        }
+
+        void stabilise(long stabiliationDurationMillis, long delayVariability) {
```

Comment:
I'd like to avoid calling `stabilise` with a different delay variability if possible - could we inline this parameter?

Reply:
sure, done in a7c286c